src/Uno.UI/UI/Xaml/Application.cs

```diff
@@ -199,7 +199,7 @@ private void SetRequestedTheme(ApplicationTheme requestedTheme)
 
 		private void OnRequestedThemeChanged()
 		{
-			if (Windows.UI.Xaml.Window.Current.Content is FrameworkElement root)
+			if (GetTreeRoot() is FrameworkElement root)

```

Comment:
I'm wondering if we should end up using the newer `XamlRoot` property

Reply:
...huh ...perhaps? Never noticed that one before. Not sure if it's applicable in this context.